lib/players/computer.rb

```diff
@@ -0,0 +1,82 @@
+class Player::Computer < Player
+  def move(board)
+    #1. Make winning move
+    location = make_winning_move(board)
+
+    #2. Block winning move
+    location = block_winning_move(board) if location.nil?
```

Comment:
I think you could use `||=` here to the same effect.
```
location ||= block_winning_move(board)
```
The assignment will only occur if `location` is nil.